Programmers/231005_리코쳇_로봇/kwaksj329.py

```diff
@@ -0,0 +1,33 @@
+from collections import deque
+
+def fourPosition(point, board, index):
+    move = [[-1, 0], [1, 0], [0, -1], [0, 1]]
+    while True:
+        point[0] += move[index][0]
+        point[1] += move[index][1]
+        if point[0] < 0 or point[1] < 0 or point[0] >= len(board) or point[1] >= len(board[0]): break
+        if board[point[0]][point[1]] == 'D': break
+    return point[0] - move[index][0], point[1] - move[index][1]
+            
+def solution(board):
+    answer = 0
+    visited = [[-1 for _ in range(len(board[0]))] for _ in range(len(board))]
```

Comment:
```suggestion
    visited = [[-1] * len(board[0]) for _ in range(len(board))]
```
이렇게 해도 됩니다! 
저는 코드를 더 간단하게 하기 위해서 `M, N = len(board), len(board[0])` 이런 식으로 변수에 길이를 할당하고 시작합니다.
```suggestion
    visited = [[-1] * N for _ in range(M)]
```
이외에도 지금 파일 전반에 걸쳐 `len(board)`와 `len(board[0])`이 계속 있는데, 이걸 변수에 할당해놓으면 쓰기도 읽기도 간단하겠죠!
`M`과 `N`이 헷갈린다면 `R`, `C`같은 변수명도 좋습니다.